### PR TITLE
updated the steps for WHfB

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8098

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 1]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
 
 Sign-in a workstation with access equivalent to a _domain user_.
 
-1. Sign-in to the [Azure Portal](https://portal.azure.com/).
-2. Select **All Services**.  Type **Intune** to filter the list of services.  Click **Microsoft Intune**.
-3. Click **device enrollment**.
+1. Sign-in to the [Microsoft Endpoint Manager admin center](https://endpoint.microsoft.com/).
+2. Select **Devices**.
+3. Click **Enroll devices**.
 4. Click **Windows enrollment**
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:307]:
```suggestion
4. Click **Windows enrollment**.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 2]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
...
 5. Under **Windows enrollment**, click **Windows Hello for Business**.
-   ![Create Intune Windows Hello for Business Policy](images/aadj/IntuneWHFBPolicy-00.png)
-6. Under **Priority**, click **Default**. 
-7. Under **All users and all devices**, click **Settings**.
-8. Select **Enabled** from the **Configure Windows Hello for Business** list.
-9. Select **Required** next to **Use a Trusted Platform Module (TPM)**.  By default, Windows Hello for Business prefers TPM 2.0 or falls backs to software. Choosing **Required** forces Windows Hello for Business to only use TPM 2.0 or TPM 1.2 and does not allow fall back to software based keys.
-10. Type the desired **Minimum PIN length** and **Maximum PIN length**.
+   ![Create Windows Hello for Business Policy](images/aadj/MEM.png)
+6. Select **Enabled** from the **Configure Windows Hello for Business** list.
+7. Select **Required** next to **Use a Trusted Platform Module (TPM)**.  By default, Windows Hello for Business prefers TPM 2.0 or falls backs to software. Choosing **Required** forces Windows Hello for Business to only use TPM 2.0 or TPM 1.2 and does not allow fall back to software based keys.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:311]:
```suggestion
7. Select **Required** next to **Use a Trusted Platform Module (TPM)**.  By default, Windows Hello for Business prefers TPM 2.0 or falls backs to software. Choosing **Required** forces Windows Hello for Business to only use TPM 2.0 or TPM 1.2 and does not allow fall back to software-based keys.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 3]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
...
+8. Type the desired **Minimum PIN length** and **Maximum PIN length**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:312]:
```suggestion
8. Enter the desired **Minimum PIN length** and **Maximum PIN length**.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 4]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
...
     > [!IMPORTANT]
     > The default minimum PIN length for Windows Hello for Business on Windows 10 is 6.  Microsoft Intune defaults the minimum PIN length to 4, which reduces the security of the user's PIN.  If you do not have a desired PIN length, set the minimum PIN length to 6.
 
-![Intune Windows Hello for Business policy settings](images/aadj/IntuneWHFBPolicy-01.png)
-
-11. Select the appropriate configuration for the following settings.
+9. Select the appropriate configuration for the following settings.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:316]:
```suggestion
9. Select the appropriate configuration for the following settings:
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 5]
```diff
@@ -326,10 +322,10 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
 
-12. Select **Yes** next to **Allow biometric authentication** if you want to allow users to use biometrics (fingerprint and/or facial recognition) to unlock the device.  To further secure the use of biometrics, select **Yes** to **Use enhanced anti-spoofing, when available**.
-13. Select **No** to **Allow phone sign-in**.  This feature has been deprecated.
-14. Click **Save**
-15. Sign-out of the Azure portal.
+10. Select **Yes** next to **Allow biometric authentication** if you want to allow users to use biometrics (fingerprint and/or facial recognition) to unlock the device.  To further secure the use of biometrics, select **Yes** to **Use enhanced anti-spoofing, when available**.
+11. Select **No** to **Allow phone sign-in**.  This feature has been deprecated.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:326]:
```suggestion
11. Select **No** to **Allow phone sign-in**. This feature has been deprecated.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 6]
```diff
@@ -326,10 +322,10 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
+12. Click **Save**
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:327]:
```suggestion
12. Choose **Save**
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 7]
```diff
@@ -326,10 +322,10 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
+13. Sign-out of the Microsoft Endpoint Manager admin center.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:328]:
```suggestion
13. Sign out of the Microsoft Endpoint Manager admin center.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 8]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
 
 Sign-in a workstation with access equivalent to a _domain user_.
 
-1. Sign-in to the [Azure Portal](https://portal.azure.com/).
-2. Select **All Services**.  Type **Intune** to filter the list of services.  Click **Microsoft Intune**.
-3. Click **device enrollment**.
+1. Sign-in to the [Microsoft Endpoint Manager admin center](https://endpoint.microsoft.com/).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:304]:
```suggestion
1. Sign in to the [Microsoft Endpoint Manager admin center](https://endpoint.microsoft.com/).
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 9]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
...
+2. Select **Devices**.
+3. Click **Enroll devices**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:306]:
```suggestion
3. Choose **Enroll devices**.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md (hunk 1 above)
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:307]:
```suggestion
4. Select **Windows enrollment**
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 10]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
...
 4. Click **Windows enrollment**
 5. Under **Windows enrollment**, click **Windows Hello for Business**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:308]:
```suggestion
5. Under **Windows enrollment**, select **Windows Hello for Business**.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 11]
```diff
@@ -301,23 +301,19 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
...
-9. Select **Required** next to **Use a Trusted Platform Module (TPM)**.  By default, Windows Hello for Business prefers TPM 2.0 or falls backs to software. Choosing **Required** forces Windows Hello for Business to only use TPM 2.0 or TPM 1.2 and does not allow fall back to software based keys.
-10. Type the desired **Minimum PIN length** and **Maximum PIN length**.
+   ![Create Windows Hello for Business Policy](images/aadj/MEM.png)
+6. Select **Enabled** from the **Configure Windows Hello for Business** list.
+7. Select **Required** next to **Use a Trusted Platform Module (TPM)**.  By default, Windows Hello for Business prefers TPM 2.0 or falls backs to software. Choosing **Required** forces Windows Hello for Business to only use TPM 2.0 or TPM 1.2 and does not allow fall back to software based keys.
+8. Type the desired **Minimum PIN length** and **Maximum PIN length**.
     > [!IMPORTANT]
     > The default minimum PIN length for Windows Hello for Business on Windows 10 is 6.  Microsoft Intune defaults the minimum PIN length to 4, which reduces the security of the user's PIN.  If you do not have a desired PIN length, set the minimum PIN length to 6.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:314]:
```suggestion
    > The default minimum PIN length for Windows Hello for Business on Windows 10 is six.  Microsoft Intune defaults the minimum PIN length to four, which reduces the security of the user's PIN.  If you do not have a desired PIN length, set the minimum PIN length to six.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 12]
```diff
@@ -326,10 +322,10 @@ Sign-in a workstation with access equivalent to a _domain user_.
...
 
-12. Select **Yes** next to **Allow biometric authentication** if you want to allow users to use biometrics (fingerprint and/or facial recognition) to unlock the device.  To further secure the use of biometrics, select **Yes** to **Use enhanced anti-spoofing, when available**.
-13. Select **No** to **Allow phone sign-in**.  This feature has been deprecated.
-14. Click **Save**
-15. Sign-out of the Azure portal.
+10. Select **Yes** next to **Allow biometric authentication** if you want to allow users to use biometrics (fingerprint and/or facial recognition) to unlock the device.  To further secure the use of biometrics, select **Yes** to **Use enhanced anti-spoofing, when available**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:325]:
```suggestion
10. Select **Yes** next to **Allow biometric authentication** if you want to allow users to use biometrics (fingerprint and/or facial recognition) to unlock the device. To further secure the use of biometrics, select **Yes** to **Use enhanced anti-spoofing, when available**.
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 13]
```diff
@@ -326,10 +322,10 @@ Sign-in a workstation with access equivalent to a _domain user_.
     > [!NOTE]
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:322]:
```suggestion
    
    > [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**